cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -2916,13 +2895,10 @@
             "Ref": "MaxSize"
           },
           "DynamoDBTable": {
-            "Ref": "DynamoDBTable"
+            "Ref": "AWS::StackName"
           },
           "SQSQueueName": {
```

Comment:
Should we have a condition?

Reply:
Not really needed. When `awsbatch` the table is not created nor it is used